Shopkeeper/Shopkeeper.Services/Interfaces/IItemService.cs

```diff
@@ -8,7 +8,7 @@ public interface IItemService
     {
         Item Get(string itemId);
         IQueryable<Item> GetAll();
```

Comment:
Might consider not using IQueryable when you aren't making roundtrips to any datastore. I think IEnumerable is less surprising